packages/system/src/components/homepage/hero.tsx

```diff
@@ -52,21 +52,21 @@ export function Hero({ className, siteName, ...props }: HeroProps) {
 			<div className={heroFeatureHighlightRowStyle}>
 				<a
 					className={heroFeatureHighlightStyle}
-					href="https://react.framework.dev/categories/libraries"
+					href="/categories/libraries"
```

Comment:
While this is a good fix - I don't think we actually want these elements to be linkable so maybe best to just get rid of the anchor altogether!

Reply:
Sure! But just to be sure, this was added because this feature request: https://github.com/thisdot/framework.dev/issues/189, is this change not required anymore?

Reply:
Oh... uh... I didn't realize that was intentional. Ignore this one then. All good on the fix!